python/lsst/meas/algorithms/matcherSourceSelector.py

```diff
@@ -162,4 +177,38 @@ def _isUsable(self, source):
             and not source.get(self.fluxFlagKey) \
             and self._goodSN(source)
 
+    def _isUsable_vector(self, sourceCat):
```

Comment:
Unfortunately Python does not detect duplicate methods; it silently picks one.

Thanks for running a linter, though I hope it was flake8, not pep8. The former warns of important things like undefined variables and duplicate methods. The latter only warns about pep8 violations, which are much less interesting. (Then there's autopep8 that reformats code in an attempt to make it meet pep8 standards; it's not as smart as clang-format and can't fix everything, so I recommend running flake8 and manually inspecting the results afterwards).